.github/workflows/ci.yml

```diff
@@ -14,14 +14,15 @@ jobs:
     strategy:
       matrix:
         node: [12, 14, 16]
-        npm: [6,8]
+        npm: [6, 8.5.0]
```

Comment:
Facing some [issues](https://github.com/npm/cli/issues/4664) with npm 8.6.0 currently so pinned the version to 8.5.0